### PR TITLE
Localize server countries/cities

### PR DESCRIPTION
**Pull request by bakulf**
I spent a bit of time figuring out how to localize country and city names. The solution I found is:
- let's use the testing framework to retrieve the list of servers and languages.
- for each country/city, let's run a SPARQL query to wikidata and let's retrieve the list of localized names
- The result of these steps is a JSON object like this: ```[ { countryCode: "it", languages: { it: "Italia", fr: "Italie" } ... cities: [ ...```
- this JSON object is included in the app as a QRC resource.
- `VPNLocalizer` has 2 new methods: `translateServerCity` and `translateServerCountry` to use the JSON object to localize the names.

### CODE REVIEW

**Review by flodolo — COMMENTED** [review]
I've looked at the JSON, and there's some weird stuff going on. For example, take a  look at:
- Melbourne
- Brussels
- Frankfurt
- nl

The impression is that most translations refer to a previous version of the *en* text.

How do we go about fixing these? Do we expect people to fix them on Wikidata (I have no clue about the approval process, but I don't expect it to be easy), can the in-tree JSON file be out of sync compared to Wikidata?

Note that all US cities lose the state in most languages.

I'd also like to understand how UX feels about having mixed language in this list, because that's what is likely to happen.

nit: It would be great to have key alphabetically order ;-)

**Comment by bakulf** [comment]
>     * Melbourne

These seem to be fine. Melbourne is translated in this way:

```
ru | "Мельбурн"
de | "Melbourne City"
en | "City of Melbourne"
fr | "Ville de Melbourne"
it | "Città di Melbourne"
ja | "メルボルン"
sq | "Melburni"
zh_TW | "墨爾本"
uk | "Мельбурн"
el | "Μελβούρνη"
```

>     * Brussels

```
de | "Region Brüssel-Hauptstadt"
el | "Περιοχή των Βρυξελλών"
fr | "Région de Bruxelles-Capitale"
nl | "Brussels Hoofdstedelijk Gewest"
ru | "Брюссельский столичный регион"
hsb | "Brüssel"
sq | "Brukseli"
oc | "Region de Brussèlas Capitala"
en | "City of Brussels"
it | "Regione di Bruxelles-Capitale"
uk | "Брюссельський столичний регіон"
co | "Regione di Bruxelles-Capitale"
ja | "ブリュッセル首都圏地域"
zh_TW | "布魯塞爾-首都大區"
pt_BR | "Bruxelas"
sk | "Región Brusel-hlavné mesto"
en_CA | "City of Brussels"
en_GB | "City of Brussels"
```

>     * Frankfurt

```
co | "Francuforte su l'Oder"
nl | "Frankfurt (Oder)"
de | "Frankfurt (Oder)"
en | "Frankfurt (Oder)"
hsb | "Frankobrod nad Wódru"
it | "Francoforte sull'Oder"
ja | "フランクフルト・アン・デア・オーダー"
cy | "Frankfurt an der Oder"
dsb | "Frankobrod-Odra"
el | "Φρανκφούρτη"
fr | "Francfort-sur-l'Oder"
oc | "Frankfurt (Oder)"
ru | "Франкфурт-на-Одере"
sk | "Frankfurt nad Odrou"
sq | "Frankfurti mbi Main"
uk | "Франкфурт-на-Майні"
pt_BR | "Frankfurt an der Oder"
zh_TW | "美茵河畔法蘭克福"
```

> How do we go about fixing these? Do we expect people to fix them on Wikidata (I have no clue about the approval process, but I don't expect it to be easy), can the in-tree JSON file be out of sync compared to Wikidata?

I would prefer adding exceptions.

File: translations/servers.json [hunk 1]
```diff
@@ -0,0 +1,1855 @@
+[
+ {
+  "countryCode": "au",
+  "languages": {
+   "cy": "Awstralia",
```
Comment (flodolo) [inline translations/servers.json:5]: Looks like all the cities have now the translation for the code.

File: translations/servers.json [hunk 2]
```diff
@@ -0,0 +1,1553 @@
...
+   "sk": "Austrália",
+   "de": "Australien"
+  },
+  "cities": [
+   {
+    "city": "Melbourne",
+    "languages": {
+     "ru": "Мельбурн",
+     "de": "Melbourne City",
+     "en": "City of Melbourne",
```
Comment (flodolo) [inline translations/servers.json:26]: We should fix the ones that still have "City of"

File: translations/servers.json [hunk 3]
```diff
@@ -0,0 +1,1553 @@
...
+   "ja": "ベルギー",
+   "nl": "België",
+   "oc": "Belgica"
+  },
+  "cities": [
+   {
+    "city": "Brussels",
+    "languages": {
+     "cy": "Dinas Brwsel",
+     "en_CA": "City of Brussels",
```
Comment (flodolo) [inline translations/servers.json:111]: Same here, we should fix the "City of" variants

File: translations/servers.json [hunk 4]
```diff
@@ -0,0 +1,1553 @@
...
+     "fr": "Francfort-sur-le-Main",
+     "hsb": "Frankobrod nad Mohanom",
+     "it": "Francoforte sul Meno",
+     "ja": "フランクフルト・アム・マイン"
+    }
+   }
+  ]
+ },
+ {
+  "countryCode": "hk",
```
Comment (flodolo) [inline translations/servers.json:508]: This one, for example, doesn't have a `en` name. How do you fall back to English?

Comment (bakulf) [reply translations/servers.json:508]: The fallback is what we receive from mullvad.

File: translations/servers.json (hunk 3 above)
Comment (bakulf) [reply translations/servers.json:111]: This is confusing on wikipedia too. There 2 bruxelles entities:
- https://en.wikipedia.org/wiki/Brussels
- https://en.wikipedia.org/wiki/City_of_Brussels

I'm pointing to the second one. Do you suggest the first one?

Comment (flodolo) [reply translations/servers.json:111]: Yes, the city should be the first one (capital of the municipality).

Comment (flodolo) [reply translations/servers.json:115]: Sorry, I misread. 1st is region, 2nd is city, so we should use the 2nd, and fix manually where necessary.

**Review by lesleyjanenorton — APPROVED** [review]
Nice!! 
Screen Shot 2021-04-06 at 10 14 27 AM
Screen Shot 2021-04-06 at 10 15 07 AM

**Review by flodolo — APPROVED** [review]

**Pull request merged**